### PR TITLE
Multipathing when BOOT_OVER_SAN=y

### PR DESCRIPTION
**Pull request by schabrolles**
Force Loading multipath modules when `BOOT_OVER_SAN` is True.
Before this changes, multipath module was loaded only if multipath devices were present in `LAYOUT_FILE`. 
So, if you migrate from virtual machine (non-multipath) to `BOOT_OVER_SAN` system (usually multipathed), multipath is not loaded during recover/migration.

I propose to automatically the load of multipath modules during recovery when `BOOT_OVER_SAN` is True.

I also add `/lib*/multipath` which where missing when creating rescue image with `BOOT_OVER_SAN=y`

### CODE REVIEW

File: usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # 240_include_multipath_tools.sh
 # Boot Over SAN executables and other goodies
 
-[[ $BOOT_OVER_SAN != ^[yY1] ]] && return
+if ! is_true "$BOOT_OVER_SAN" ; then
```
Comment (gdha) [inline usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh]: Better use `if  is_false`  instead of `if ! is_true`, no?

Comment (schabrolles) [inline usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh]: I just wonder what happen if `BOOT_OVER_SAN` was not set ... will it be treated as False ?

**Comment by schabrolles** [comment]
@gdha @jsmeix I'm going to close this Pull request and re-open it.
I've added some other commit not related to this particular issue, sorry for that. 
So I prefer to separate them in separated issue in order to discuss with you about them. 

I worked a lot this weekend on solving some multipathing issue (especially during migration on different system). I'm gonna push those pull request after finishing my migration testing.

- sles11 (non-multipath) on System A <=> sles11 multipathed on System B
- sles12 (non-multipath) on System A <=> sles12 multipathed on System B
- rhel7  (non-multipath) on System A <=> rhel7 multipathed on System B
- rhel6  (non-multipath) on System A <=> rhel6 multipathed on System B
- ubuntu16.04 (non-multipath) on System A <=> ubuntu16.04 multipathed on System B

File: usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh (hunk 1 above)
Comment (jsmeix) [inline usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh]: In general:
Caution with 'is_false' versus '! is_true'!
Both tests are not identical, see the comments
for the is_true and is_false functions in
lib/global-functions.sh and see also
https://github.com/rear/rear/pull/1212#issuecomment-283698753

**Pull request closed without merging**